tests/test_clic2020.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+import shutil
+
+import pytest
+
+from neuralcompression.data import CLIC2020
+
+
+@pytest.fixture(autouse=True)
+def remove_data_directory():
+    yield
+
+    if pathlib.Path("data").exists():
+        shutil.rmtree("data")
```

Comment:
No need for this - `pytest` has a temporary data directory. See https://github.com/facebookresearch/NeuralCompression/blob/e27c96287cdf06013f38df8913e08c7b8c09e42e/tests/test_kodak.py#L27-L48